alphastats/gui/utils/llm_helper.py

```diff
@@ -80,3 +83,75 @@ def llm_connection_test(
 
     except Exception as e:
         return str(e)
+
+
+# Unused now, but could be useful in the future
+def get_display_available_uniprot_info(regulated_features: list) -> dict:
```

Comment:
unused code is dead code ;-) If you see this being used next week, it might be fine to keep, otherwise I would remove it (use a speaking commit message such that you'll find it again)